draft-ietf-dnsop-svcb-httpssvc.md

```diff
@@ -590,6 +590,26 @@ such as {{!HappyEyeballsV2=RFC8305}}.
 Some important optimizations are discussed in {{optimizations}}
 to avoid additional latency in comparison to ordinary AAAA/A lookups.
 
+## Handling resolution failures
+
+If an SVCB query results in a SERVFAIL error, transport error, or timeout,
+and DNS exchanges between the client and the recursive resolver are
+cryptographically protected (e.g. using TLS {{!RFC7858}} or HTTPS
+{{!RFC8484}}), the client MUST abandon the connection attempt and display
```

Comment:
Can the client use a previous stale answer if it has one in-cache still?

Reply:
The client may also wish to retry the query against another equally-trusted resolver?  (eg, if it has multiple resolver IPs.)

Reply:
Good point, we don't necessarily want to rule out those options.  I've changed the text from "MUST abandon" to "MUST NOT fall back".